packages/osl-operator-bundle-image/resources/config/manifests/bases/logic-operator-rhel8.clusterserviceversion.yaml

```diff
@@ -295,5 +299,5 @@ spec:
       name: IMAGE_LOGIC_SWF_BUILDER
     - image: registry.redhat.io/openshift4/ose-kube-rbac-proxy@sha256:4564ca3dc5bac80d6faddaf94c817fbbc270698a9399d8a21ee1005d85ceda56
       name: IMAGE_KUBE_RBAC_PROXY
-  replaces: logic-operator-rhel8.v1.34.0
+  replaces: logic-operator-rhel8.v0.0.0
```

Comment:
I'm now a bit confused, who will now change this to logic-operator-rhel8.v1.35.0 you @ricardozanini or @saumya1singh in follow-up PR?

Reply:
btw, my comment is not a blocker